go/vt/sqlparser/sql.y

```diff
@@ -84,6 +84,7 @@ func markBindVariable(yylex yyLexer, bvar string) {
   ctes          []*CommonTableExpr
   order         *Order
   limit         *Limit
+  rowAlias      RowAlias
```

Comment:
Is there some convention around when to use a pointer vs a value here?

Reply:
Generally if it's optional, a pointer is usually better to indicate the absence / presence of the alias here.

Reply:
Done.